cypress/integration/settings/create_dunning_type_spec.js

```diff
@@ -0,0 +1,31 @@
+import { DunningType } from '../../support/utils/dunning_type';
+
+describe('create dunning type', function() {
+  // if creating more than 1 default dunning type will give an error
+  // const timestamp = new Date().getTime();
+  // const dunningTypeName = `dunning test ${timestamp}`;
+
+  // before(function() {
+  //   cy.fixture('settings/dunning_type.json').then(dunningType => {
+  //     Object.assign(new DunningType(), dunningType)
+  //       .setName(dunningTypeName)
+  //       // .setCheckDefault(dunningType.default)
+  //       .apply();
+  //   });
+  // });
+
+  it('operations on BP', function() {
+    cy.visit('/window/123');
+
+    cy.get(':nth-child(1) > .text-left.td-sm').dblclick();
+    cy.selectTab('Customer');
```

Comment:
this doesn't look very stable to me..
why don't you use bpartner.js and the new setCustomerDunning method?